coalib/output/ConsoleInteraction.py

```diff
@@ -321,7 +321,7 @@ def print_affected_files(console_printer,
     :param console_printer: Object to print messages on the console.
```

Comment:
can you file a newcomer issue for line 319 and it's typos? (check if it already exists though!)